servers/display_server.h

```diff
@@ -280,10 +284,15 @@ class DisplayServer : public Object {
 		MOUSE_MODE_CAPTURED,
 		MOUSE_MODE_CONFINED,
 		MOUSE_MODE_CONFINED_HIDDEN,
+		MOUSE_MODE_MAX,
```

Comment:
Or linking it in the opposite way, since `Input` is already include here:

```
		MOUSE_MODE_VISIBLE = Input::MOUSE_MODE_VISIBLE,
		MOUSE_MODE_HIDDEN = Input::MOUSE_MODE_HIDDEN,
		MOUSE_MODE_CAPTURED = Input::MOUSE_MODE_CAPTURED,
		MOUSE_MODE_CONFINED = Input::MOUSE_MODE_CONFINED,
		MOUSE_MODE_CONFINED_HIDDEN = Input::MOUSE_MODE_CONFINED_HIDDEN,
		MOUSE_MODE_MAX = Input::MOUSE_MODE_MAX,
```

Reply:
Done!